### ISSUE TITLE
poor bachman readability with lots of relations

### ISSUE THREAD

**Issue by mrzor** [voormedia/rails-erd#217]
Greetings,

I have generated a diagram with `bin/rake erd attributes=foreign_keys,primary_keys,inheritance notation=bachman orientation=vertical`.

Even when zooming, the arrow ends will pack together, making the arrow endings hard to tell apart / decypher. 

Image worth a thousand words:
![railserd-1](https://cloud.githubusercontent.com/assets/54230/16949375/b71d6c7e-4dba-11e6-85e2-184ef24f9288.png)

Is there a way to improve on this?

**Comment by kerrizor** [comment]
Agreed! I'd like to improve the appearance of edges for sure. #256 should help, and the new splines options might improve it as well

```
overlap = false;
splines = true;
```

**Comment by kerrizor** [comment]
Closing as stale (7+ years old). If this feature is still desired, please open a new issue. Thanks! 🙏
